.github/workflows/base_node_binaries.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "name": "linux-x86_64",
+    "runs-on": "ubuntu-18.04",
+    "rust": "nightly-2022-05-01",
+    "target": "x86_64-unknown-linux-gnu",
+    "cross": false,
+    "target_cpu": "x86-64",
+    "features": "safe"
+  },
+  {
+    "name": "linux-arm64",
+    "runs-on": "ubuntu-18.04",
+    "rust": "stable",
+    "target": "aarch64-unknown-linux-gnu",
+    "cross": false,
+    "target_cpu": "generic",
+    "target_bins": "--bin tari_base_node --bin tari_console_wallet --bin tari_merge_mining_proxy --bin tari_miner --bin tari_validator_node",
+    "features": "safe"
+  },
+  {
+    "name": "macos-x86_64",
+    "runs-on": "macos-11",
+    "rust": "stable",
+    "target": "x86_64-apple-darwin",
+    "cross": false,
+    "target_cpu": "x86-64",
+    "features": "safe"
+  },
+  {
+    "name": "macos-arm64",
+    "runs-on": "macos-latest",
+    "rust": "stable",
+    "target": "aarch64-apple-darwin",
+    "cross": true,
+    "target_cpu": "generic",
+    "features": "safe"
+  },
+  {
+    "name": "windows-x64",
+    "runs-on": "windows-2019",
+    "rust": "stable",
+    "target": "x86_64-pc-windows-msvc",
+    "cross": false,
+    "target_cpu": "x86-64",
+    "features": "safe"
```

Comment:
CPU target set, if the windows binary is run on older hardware, I think we see a ```illegal cpu instruction```. We could really use the matrix build and target a faster CPU instruction set like avx2, but doubling up the builds might not add value?

Reply:
I think we generally only target Haswell and above. Anything older should compile from source.

I think we only use safe on macos because it used to fail

Reply:
```target_cpu``` and ```features``` could be matrix build in futre, but thinking we get the base in and then we can extend as needed or as requested?

Also not sure what features we want to enable in a build, but can be done per matrix set.